lang/tags/comment_block.py

```diff
@@ -1,13 +1,19 @@
-from talon import Context, Module
+from talon import Context, Module, actions
 
 ctx = Context()
 mod = Module()
 
 mod.tag("code_comment_block", desc="Tag for enabling generic block comment commands")
+mod.tag("code_comment_block_c_like", desc="Denotes usage of C-style block comments")
+
+ctx.matches = """
```

Comment:
maybe name this `c_like_ctx`, to be more explicit about what it matches?

Reply:
Done.